src/canmatrix/canmatrix.py

```diff
@@ -574,10 +617,10 @@ class Frame(object):
     """
 
     name = attr.ib(default="")
-    id = attr.ib(type=int, default=0)
+    arbitration_id = attr.ib(converter=Arbitration_Id.from_compound_integer, default=0)
     size = attr.ib(default=0)
     transmitters = attr.ib(type=list, factory=list)
-    extended = attr.ib(type=bool, default=False)
+    # extended = attr.ib(type=bool, default=False)
```

Comment:
Hmm, So we're changing from `default=False` to `default=True` effectively above.  I'm not sure which it should be, or how much it matters.

Do delete this before it gets left hanging around.